public/views/FileTransfer.svelte

```diff
@@ -368,6 +367,32 @@
 
   $: watchUsersCount(usersCount, percentage);
 
+  function dropHandler(ev) 
+  {
+    ev.preventDefault();
+    var files = ev.dataTransfer.files;
+    if (files != null)
+    {
+      document.getElementById('drop_zone').style.border = 'dotted';
+      selectFiles(files);
+    }
+  }
+  function dragOverHandler(ev) 
+  {
+    ev.preventDefault();
+    document.getElementById('drop_zone').style.border = 'solid';
+  }
+  function dragLeaveHandler(ev) 
+  {
+    ev.preventDefault();
+    document.getElementById('drop_zone').style.border = 'dotted';
+  }
+  function changeHandler(ev)
+  {
+    var files = ev.target.files;
+    selectFiles(files);
+  }
```

Comment:
I think most of this can be reduced and written inline within `drop_zone` element.
Consider reading https://svelte.dev/docs#on_element_event  (Component Events and Modifiers) in Svelte.